test/dummy/config/initializers/merit.rb

```diff
@@ -42,6 +42,9 @@
 }, {
   id: (badge_id = badge_id+1),
   name: 'gossip'
+}, {
+  id: (badge_id = badge_id+1),
```

Comment:
Useless assignment to variable - `badge_id`.<br>Surrounding space missing for operator '+'.